src/ui/components/search/searchcomponent.js

```diff
@@ -7,10 +7,14 @@ import SearchParams from '../../dom/searchparams';
 import TranslationFlagger from '../../i18n/translationflagger';
 import QueryUpdateListener from '../../../core/statelisteners/queryupdatelistener';
 import QueryTriggers from '../../../core/models/querytriggers';
+import SearchStates from '../../../core/storage/searchstates';
 
 const IconState = {
-  YEXT: 0,
-  MAGNIFYING_GLASS: 1
+  LOADING: 0,
```

Comment:
Should we have a CUSTOM_LOADING state for consistency with the CUSTOM_ICON state?

Reply:
I will make this update so `iconElClass` is not needed when requesting icon animation frame